includes/classes/missions/MissionCaseExpedition.class.php

```diff
@@ -125,7 +125,7 @@ function EndStayEvent()
 				$founded		= round(min($maxFactor, max(200, $factor)) * $fleetPoints);
 
 				$fleetColName	= 'fleet_resource_'.$resource[$resourceId];
-				$this->UpdateFleet($fleetColName, $this->_fleet[$fleetColName] + $founded);
+				$this->UpdateFleet($fleetColName, $this->_fleet[$fleetColName] + min($founded, $fleetCapacity));
 			break;
```

Comment:
Can you change the min to Line 125? I think to more readable, instead modify value in functions calls.